new_django_project/blog/api/models.py

```diff
@@ -9,3 +9,12 @@ class Post(models.Model):
     class Meta:
         ordering = ['created']
 
+class Comment(models.Model):
+    created = models.DateTimeField(auto_now_add=True)
+    body = models.TextField(blank=False)
+    owner = models.ForeignKey('auth.User', related_name='comments', on_delete=models.CASCADE)
+    post = models.ForeignKey('Post', related_name='comments', on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ['created']
+
```

Comment:
[just a 💅  suggestion]
There is this method that you can use to pretty print your object that is very useful for debugging 😄  What do you think about using it here?
```suggestion

     def __repr__(self):
         return f'(Comment by {self.owner} on post {self.post}: "{self.body}")'
```

Reply:
An example with the difference that it makes 😄 
![image](https://user-images.githubusercontent.com/17599203/143509165-d497e719-c156-409b-81c4-28323384c424.png)
Other thing that can be useful and less verbose than my suggestion to add as `__repr__` of your model is the name of the object + it's database id (it is simple and helps you to find the object in the database if necessary 😄 )